client/auth.ts

```diff
@@ -2,22 +2,23 @@
 
 import NextAuth from "next-auth";
 
+console.log(process.env.A12N_URL);
+
 export const { handlers, signIn, signOut, auth } = NextAuth({
   debug: true,
-  secret: process.env.NEXTAUTH_SECRET,
+  secret: process.env.AUTH_SECRET,
   providers: [
     {
       id: "a12n-server",
       name: "a12n-server",
       type: "oidc",
```

Comment:
So this one is `type: "oidc"`, but our draft next-auth submission is `type: "oauth"` https://github.com/curveball/next-auth/pull/1/files 

Which shall we go with?

Reply:
oicd. Until yesterday I hadn't implemented enough OpenID connect but i think enough boxes are checked!